sopel/modules/safety.py

```diff
@@ -212,80 +250,167 @@ def url_handler(bot: Sopel, trigger: Trigger):
             )
             bot.say(
                 "{} {} of {} engine{} flagged a link {} posted as malicious".format(
-                    formatting.bold(formatting.color("WARNING:", "red")),
+                    bold(color("WARNING:", colors.RED)),
                     positives,
                     total,
                     "" if total == 1 else "s",
-                    formatting.bold(trigger.nick),
+                    bold(trigger.nick),
```

Comment:
Kinda gratuitous refactoring but OK.

Reply:
It started feeling silly with `formatting.bold(formatting.color("foo bar", formatting.colors.RED))`